README.md

```diff
@@ -83,9 +99,19 @@ The easiest path for getting cython, oneDAL, MPI etc. is by creating a conda env
 ```
 conda create -n DAAL4PY python=3.7 impi-devel daal daal-include cython jinja2 numpy clang-tools -c intel -c conda-forge
 conda activate DAAL4PY
-export DAALROOT=$CONDA_PREFIX
+export DALROOT=$CONDA_PREFIX
 export MPIROOT=$CONDA_PREFIX
 ```
+For build with oneAPI support additional step requered:
+1. DPC++ compiler - DPCPPROOT variable should be defiend and pointed to DPCPP compier
+2. Intel(R) oneDAL with oneAPI support
+    a. Can be installed from Conda channel
+    b. Can be installed fron oneAPI packages repository and passed via DALROOT variable
```

Comment:
this repeats the same info as above, do you really need to repeat it?

Reply:
done